README.md

```diff
@@ -132,85 +132,86 @@ To add your own ICO you would have to:
 
 1. Fork the project - Github won't allow you to create pull request otherwise.
 
-2. Include the ICO Smart-Contract [ABI](https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI) in the [Smart_Contracts](https://github.com/Neufund/ico-transparency-monitor/tree/master/src/smart_contracts) folder
-    this ABI is generated from the smart-contract source code and can be found in some cases in [etherscan](https://etherscan.io/)
+2. Include the ICO Smart-Contract [ABI](https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI) in the [Smart_Contracts](https://github.com/Neufund/ico-transparency-monitor/tree/master/src/smart_contracts) folder, name the files with the contract address,
```

Comment:
name the files with the contract address
`the name of ABI file should be exactly like the contract address`